blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/ExportConfig.java

```diff
@@ -10,6 +12,7 @@ public class ExportConfig {
      * @deprecated
      *      Use getter and setter
      */
+    @SuppressFBWarnings(value="PA_PUBLIC_PRIMITIVE_ATTRIBUTE", justification="Preserve API compatibility")
```

Comment:
Required because the parent POM update picks up a new SpotBugs version. The justification is just copied over from https://github.com/jenkinsci/stapler/blob/35363c1920d2701aeefb05b57b07d721dfc48c5d/core/src/main/java/org/kohsuke/stapler/export/ExportConfig.java#L16. I do not know why we have a copy of half of that package here.